libs/sdk-ui-dashboard/src/model/types/commonTypes.ts

```diff
@@ -125,6 +125,13 @@ export type DashboardConfig = {
      * Optionally configure which of the default menu button buttons are visible.
      */
     menuButtonItemsVisibility?: IMenuButtonItemsVisibility;
+
+	/**
```

Comment:
Strange formatting, might be tripping up validation

Reply:
fixed